apprentice_teacher.rb

```diff
@@ -1,21 +1,10 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'faculty'
+class ApprenticeTeacher < Faculty
 
   def initialize(options={})
```

Comment:
how can you DRY this up even more with some constants and throwing the attributes into the parent class?